src/fusion-rates-gamow.md

```diff
@@ -0,0 +1,861 @@
+- Decay into a neutron and $\rm ^3He$
+
+Different $\rm D_2$ states tunnel to different $\rm ^4He^*$ states which have unique decay rates $\gamma$. The rates can be found in Table 3.0.1 (page 19) of [Tilley et.al](https://www.sciencedirect.com/science/article/abs/pii/037594749290635W).
+
+<!-- #region -->
+**For the $^5S$ state of $\rm D_2$**:
+
+We must refer to the $\rm ^5S$ $J^\pi=2^+$ state of $\rm ^4He^*$ at 27.42 MeV, whose line width is:
+
+$$
+\hbar \gamma_{^5S} ~=~ 0.25 + 0.23~\text{MeV} ~=~ 0.48~ \text{MeV}
+$$
+
+
+and whose associated decay rate and lifetime is:
+
+$$
+\gamma_{^5S} ~=~ {0.48 ~ {\rm MeV} \over \hbar} ~=~ 7.3 \times 10^{20}~{\rm s}^{-1}
+$$
+$$
+\tau_{^5S} ~=~ {\hbar \over 0.48 ~ {\rm MeV} } ~=~ 1.4 \times 10^{-21} ~{\rm sec} 
+$$
+
+We don't have data for the $^1S$ state, so we'll use the $^5S$ result.
+<!-- #endregion -->
+
+<!-- #region -->
+**For the $^3P$ state of $\rm D_2$**:
+
+We must refer to the $\rm ^3P$ $J^\pi=1^-$ state of $\rm ^4He^*$ at 28.37 MeV, whose line width is:
+
+$$
+\hbar \gamma_{^3P} ~=~ 0.07 + 0.08~\text{MeV} ~=~ 0.15~ \text{MeV}
+$$
+
+
+and whose associated decay rate and lifetime is:
+
+$$
+\gamma_{^3P} ~=~ {0.15 ~ {\rm MeV} \over \hbar} ~=~ 2.3 \times 10^{20}~{\rm s}^{-1}
+$$
+$$
+\tau_{^3P} ~=~ {\hbar \over 0.15 ~ {\rm MeV} } ~=~ 4.4 \times 10^{-21} ~{\rm sec} 
+$$
+<!-- #endregion -->
+
+<!-- #region -->
+**For the $^5D$ state of $\rm D_2$**:
+
+We must refer to the $\rm ^5D$ $J^\pi=2^+$ state of $\rm ^4He^*$ at 29.89 MeV, whose line width is:
+
+$$
+\hbar \gamma_{^5D} ~=~ 0.04 + 0.04~\text{MeV} ~=~ 0.08~ \text{MeV}
+$$
+
+
+and whose associated decay rate and lifetime is:
+
+$$
+\gamma_{^5D} ~=~ {0.08 ~ {\rm MeV} \over \hbar} ~=~ 1.2 \times 10^{20}~{\rm s}^{-1}
+$$
+$$
+\tau_{^5D} ~=~ {\hbar \over 0.08 ~ {\rm MeV} } ~=~ 8.2 \times 10^{-21} ~{\rm sec} 
+$$
+
+<!-- #endregion -->
+
+```python
+for i, gamma in enumerate([7.3e20, 7.3e20, 2.3e20, 1.2e20]):
+    states[i]["gamma"] = gamma
+    gamow_74.loc[i,"$\gamma$"] = gamma
+
+gamow_74
+```
+
+## Calculating the fusion rates for 74 pm separation
+
+
+Now we have everything we need to calculate the fusion rates via
+
+$$\Gamma = \frac{v_{nuc}}{v_{mol}}e^{-2G}\gamma$$
+
+```python
+gamow_74["$\Gamma$"] = gamow_74["$v_{nuc}/v_{mol}$"]*gamow_74["$e^{-2G}$"]*gamow_74["$\gamma$"]
+gamow_74
+```
+
+It's instructive to compare the Gamow based fusion rates to those calculated by solving the Schrödinger equation fully. One of the earliest calculations was done by [Koonin and Nauenberg in 1989](https://www.nature.com/articles/339690a0). They calculated a fusion rate for an $S$ state using the molecular potential and neglecting the nuclear potential. 
+
+Their rate was calculated to be $\Gamma = 3 \times 10^{-64} \, \rm s^{-1}$
+
+Although the Gamow rate is 3 orders of magnitude below the more accurate value, the Gamow approach allows us to incorporate screening effects more easily. We'll see this later in the notebook.
```

Comment:
Suggestion: "...screening effects more easily, so this approach has its own set of benefits."